packages/react-utilities/src/hooks/useOnClickOutside.ts

```diff
@@ -15,6 +15,13 @@ export type UseOnClickOutsideOptions = {
    */
   callback: (ev: MouseEvent | TouchEvent) => void;
 
+  /**
+   * By default uses element.contains, but custom contain function can be provided
+   * @param parentRef provided parent ref
+   * @param child event target element
+   */
+  contains?(parentRef: React.MutableRefObject<HTMLElement | undefined | null>, child: HTMLElement): boolean;
```

Comment:
```suggestion
  contains?(parent: HTMLElement | undefined, child: HTMLElement): boolean;
```

Can we pass `HTMLElement` there instead of `.Ref`?

Reply:
if you're OK with `contains` as an option in general, then I will make a separate PR to change the implem of the hook

Reply:
OH I see, we just want to modify the signature, that's fine